osmaxx-py/osmaxx/job_progress/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from osmaxx.job_progress import views
+
+urlpatterns = [
+    url(r'^tracker/(?P<order_id>[0-9]+)$', views.tracker, name='tracker'),
```

Comment:
it is quite usual to have a trailing slash in the django urls ([though this can be changed](https://docs.djangoproject.com/en/dev/ref/settings/#append-slash)). 
I think we should stick with the default here.